### ISSUE TITLE
[QR Code Scanner] No response after screen capture

### ISSUE THREAD

**Issue by stevenwdhu** [raycast/extensions#6178]
### Extension

https://www.raycast.com/StevenRCE0/qr-code-scanner

### Description

When activating the extension, the screen capture is normally started. I checked the /tmp directory and the screen shot is created. But after capturing the screen, nothing happened, and there was no error message displayed.

### Steps To Reproduce

1. activate the extension
2. capture the screen
3. see nothing happens

### Current Behaviour

no result after capturing the screen

### Expected Behaviour

output the captured QR code's URL

**Comment by StevenRCE0** [comment]
I wasn't able to reproduce the bug. Could you clone the repo, cd into

`extension/qr-code-scanner`

then run

`npm install`

followed by

`npm run dev`

to see if there's anything suspicious?

**Comment by stevenwdhu** [comment]
I tried to follow these steps to run QR Code Scanner in dev mode.
<img width="557" alt="image" src="https://user-images.githubusercontent.com/30546031/236360760-12b9fb7b-a0c1-49dd-8c24-d415daf08f02.png">

There was a dev extension in my Raycast and running it I got this printed on my terminal. There was still no response after capturing the screen.
<img width="849" alt="image" src="https://user-images.githubusercontent.com/30546031/236360596-ad938c3d-48af-4fb1-b44d-4c18d168154d.png">
